src/process.cr

```diff
@@ -36,14 +36,21 @@ class Process
   end
 
   # Sends a *signal* to the processes identified by the given *pids*.
+  @[Deprecated("Use #signal instead")]
   def self.kill(signal : Signal, *pids : Int)
     pids.each do |pid|
-      ret = LibC.kill(pid, signal.value)
-      raise Errno.new("kill") if ret < 0
+      signal(signal, pid)
     end
     nil
   end
 
+  # Sends a *signal* to the process identified by the given *pid*.
+  def self.signal(signal : Signal, pid : Int)
+    ret = LibC.kill(pid, signal.value)
+    raise Errno.new("kill") if ret < 0
+    nil
```

Comment:
This is fine, but it's *slightly* cleaner to just add an explicit `: Nil` return type to the function. The compiler will automatically ignore the return value of the function in that case, and return `nil`.

Reply:
ok, done